src/interactions/hoverInteraction.ts

```diff
@@ -45,14 +46,20 @@ export module Interaction {
 
     public _anchor(component: Hoverable, hitBox: D3.Selection) {
       super._anchor(component, hitBox);
-      this._dispatcher = new Dispatcher.Mouse(this._hitBox);
+      this._dispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> (<any> this._componentToListenTo)._element.node());
```

Comment:
`<any>` casting to get `_element`.

The other casting is because `<Element>` can't be cast to `<SVGElement>` directly, and `.node()` returns `Element` (regardless of what the node actually is). I could add a comment about the cast, which would look mysterious after compilation to js because all the casts vanish.